app/src/main/res/layout/pin_entry_card.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge
+  tools:parentTag="android.support.v7.widget.CardView"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <ProgressBar
+    android:id="@+id/pinentry_progress"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:visibility="invisible" />
+
+  <LinearLayout
+    android:id="@+id/pinentry_pin_container"
+    style="@style/Clinic.CardContentContainer.AppLock">
```

Comment:
There are styles in this layout that are inconsistent with the usage: `LoginPinLabel` and `Clinic.CardContentContainer.AppLock` for example.

Should we rename them to be more appropriate?

Reply:
Yes, thanks for catching this.

Reply:
Don't think I can change `LoginPinLabel` or `LoginPinErrorText`. They're being used in other places too.

Reply:
`Clinic.CardContentContainer.AppLock` isn't shared with any other screen so I'm removing it.

Reply:
Update: False alarm. `Clinic.CardContentContainer` is still being used so I'm renaming `Clinic.CardContentContainer.AppLock` to `Clinic.CardContentContainer.PinEntry`